index.html

```diff
@@ -189,6 +200,9 @@ <h2 class="compliance-font uppercased-unsized" align="center" style="width: 200p
   <script src="https://cdn.jsdelivr.net/npm/vue-prism-editor@1.3.0/dist/prismeditor.umd.production.min.js"></script>
   <script src="https://cdn.jsdelivr.net/npm/prismjs@1.25.0"></script>
 
+  <script src="https://unpkg.com/v-tooltip@2.1.3/dist/v-tooltip.min.js"></script>
```

Comment:
Use the same content delivery network for libraries (same as prismeditor and prismjs